app/javascript/controllers/interview_report_form_controller.js

```diff
@@ -2,14 +2,19 @@ import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
   static get targets () {
-    return [ "togglable" ]
+    return [ "analytics", 'answers' ]
   }
 
-  showTogglable() {
-    this.togglableTarget.classList.remove('d-none')
-  }
-
-  hideTogglable() {
-    this.togglableTarget.classList.add('d-none')
+  toggle(event) {
```

Comment:
@harold93 SRP the toggle function should not have to care about this initial state of the page.
3 items to update ==> for each case (data, classic or answers) we want to control all 3 items.